.github/workflows/test.yaml

```
@@ -5,6 +5,8 @@ on:
     branches: [main]
   pull_request:
     branches: [main]
+  schedule:
+    - cron: "0 5 */14 * *"
```

```suggestion
    - cron: "0 5 1,15 * *"
```

This would be a better bi-weekly: <https://crontab.guru/#0_5_1,15_*_*>

> next at 2023-06-15 05:00:00
> then at 2023-07-01 05:00:00
> then at 2023-07-15 05:00:00
> then at 2023-08-01 05:00:00
> then at 2023-08-15 05:00:00